app/controllers/runs_controller.rb

```diff
@@ -83,7 +83,7 @@ def update
 
     if params[@run.id36][:video_url]
       if @run.update(video_url: params[@run.id36][:video_url])
-        redirect_to edit_run_path(@run), notice: 'Proof saved.'
+        redirect_back(fallback_location: edit_run_path(@run), notice: 'Video saved! 📹')
```

Comment:
I think we should just leave them around to get cleaned up later. I don't see any advantage to introducing more code to clean them up sooner. 

From the user perspective, it doesn't really matter since the "Auto-Highlight Possible" button hides when there's a video URL present on the run. The only difference is whether it appears again if you later clear the video URL, which I don't think is really a big deal either way. So let's go with the simpler code option.